pages/infrastructure_providers-validators/optimize_full_node.md

```diff
@@ -0,0 +1,78 @@
+# Optimize Your Full Node
+Optimizing your full node helps keep it online, up to date, and operating quickly. Faster nodes have an advantage over slower nodes because they tend to receive new data first and they minimize the time between placing and resolving orders. Optimize your full node by connecting to trusted nodes, taking precautions against falling out of sync with the network, and configuring storage settings.
+
+> Code snippets on this page use example values. Replace them with your own. See the [Network Configuration](../infrastructure_providers-network/network_constants.mdx) section of the documentation for network constants and other resources you need to configure a full node.
+
+## Prerequisites
+You need a running, non-validating full node that is connected to a network. 
+
+- If you created a system service for your node by following the instructions on the previous page, [Set Up a Full Node](../infrastructure_providers-validators/set_up_full_node.md), start your node with the following command:
+  ```bash
+  stystemctl start dydxprotocold
```

Comment:
for my understanding, does this command not need the `--non-validating-full-node=true` flag?

Reply:
It does require the flag, but the Set Up a Full Node procedure has you set up a system service that includes the flag already. So this just calls out that if you're coming here from the Next Steps of that procedure, this is how its set up